js/ui/scheduler/ui.scheduler.subscribes.js

```diff
@@ -658,9 +658,10 @@ var subscribes = {
     },
 
     convertDateByTimezone: function(date, appointmentTimezone) {
+        var msInHour = 3600000;
```

Comment:
Maybe it would better to extract it to the top level as the constant and replace all occurrences in the file with the constant instead of "magic number"?